tests/unit.suite.yml

```diff
@@ -3,6 +3,9 @@
 # Suite for unit (internal) tests.
 
 class_name: UnitTester
+coverage:
+    include:
+        - src/commands/*
```

Comment:
why only this folder?

Reply:
wouldn't it be simpler to have all the folders included in one place only? What does it change to do it like this? Except lower the coverage percent but we don't care because the one that counts is when all three are merged.